wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -0,0 +1,103 @@
+# osu! wiki maintenance
+
+*See also: [osu! wiki contribution guide](/wiki/osu!_wiki_Contribution_Guide)*
+
+This article describes technical or administrative aspects of the osu! wiki. It also covers maintenance procedures, which are required to keep it fresh—perhaps you may [help with one of them](#routines). For all discussions related to the wiki, use the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server).
+
+## Administrators
+
+*Main article: [List of the osu! wiki maintainers](/wiki/osu!_wiki/Maintenance/List_of_maintainers)*
```

Comment:
We use main page currently: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#main-page

But I agree it should follow wikipedia's "main article" as it conflicts the wiki's main page.
```suggestion
*Main page: [List of the osu! wiki maintainers](/wiki/osu!_wiki/Maintenance/List_of_maintainers)*
```